plio/io/io_bae.py

```diff
@@ -1,88 +1,133 @@
+import json
```

Comment:
#This file contains method to read and write various BAE(tm) file formats
#specifically to support their photogrammtery software Socet SET and Socet GXP

Reply:
In reviewing another PR (https://github.com/USGS-Astrogeology/PDS-Pipelines/pull/7) I commented that I thought that we should do a short get together on python docos and packaging - I think this comment 100% belongs in the documentation for this module.